src/three/plugins/ImplicitTilingPlugin.d.ts

```diff
@@ -0,0 +1,3 @@
+export class ImplicitTilingPlugin {
+	constructor();
+}
```

Comment:
Is it necessary to provide a constructor definition here? I'm not a typescript expert so I'm not sure. Is this enough?

```ts
export class ImplicitTilingPlugin {}
```